src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/NomisApiServiceTest.kt

```diff
@@ -54,7 +55,7 @@ internal class NomisApiServiceTest {
     }
 
     @Test
-    internal fun `when offender is not found an exception is thrown`() {
+    fun `when offender is not found an exception is thrown`() {
```

Comment:
curious, why you removing the `internal` ?

Reply:
its not a big deal, its just that a weird function name is shown when running the tests

Reply:
It also confuses the Intellij test runner which can't see the test (or is that when nested inside 2 classes?)

Reply:
OK